fix: remove vlan and hardcoded interface type (#11)
netboxlabs/diode-agent: diode-napalm-agent/README.md

Variables (using `${ENV}` syntax) can be referenced in the configuration file from environmental variables or from a provided `.env` file.

> The `driver` device attribute is optional. If not specified, the agent will attempt to find a match from NAPALM supported/installed drivers.

Reviewer: ```suggestion
The `driver` device attribute is optional. If not specified, the agent will attempt to find a match from NAPALM supported and installed drivers.
```

- Cisco NX-OS ("nxos")
- Juniper JunOS ("junos")

> Moreover, if a NAPALM [community driver](https://github.com/napalm-automation-community) is installed in the environment, it can be used in the agent's policy and also in automatic driver matching when none are specified.

Reviewer: ```suggestion
For NAPALM [community drivers](https://github.com/napalm-automation-community) installed in the environment, they can be referenced in the agent policy and will be used for automatic driver matching if no driver is specified.
```

### Supported Netbox Object Types

> Currently, once napalm agent connects to a network device, it tries to fetch and ingest to Diode the following Netbox Object Types:

Reviewer: ```suggestion
The Diode NAPALM agent tries to fetch information from network devices about the following NetBox object types:
```
